app/components/github_integration/comments/integration.py

```diff
@@ -96,7 +98,10 @@ async def reply_with_comments(message: dc.Message) -> None:
     )
     await message.edit(suppress=True)
     comment_linker.link(message, sent_message)
-    await remove_view_after_timeout(sent_message)
+    await asyncio.gather(
+        remove_view_after_timeout(sent_message),
```

Comment:
It doesn't matter but I kinda think it should be the other way around because the suppress timeout runs first.

```suggestion
        suppress_embeds_after_timeout(sent_message),
        remove_view_after_timeout(sent_message),
```

(I can't get the suggestion to work no matter how hard I try, thank you GitHub /s /nc.)

Reply:
This suggestion is outdated but GitHub doesn't say so for some reason.